### PR TITLE
Update kustomization.yaml

### PR DESCRIPTION
**Pull request by LavaAdventures**
Changed prefix to labelbot!

### CODE REVIEW

**Comment by Kir-Antipov** [comment]
Hello from Shitoberfest! Gfy.

File: deployment/base/kustomization.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-namePrefix: label-bot-
+namePrefix: labelbot!
 commonLabels:
```
Comment (mdc54-dot) [inline deployment/base/kustomization.yaml:4]: Hello

**Review by mdc54-dot — APPROVED** [review]

File: deployment/base/kustomization.yaml (hunk 1 above)
Comment (mdc54-dot) [reply deployment/base/kustomization.yaml:4]: deployment/base/kustomization.yaml

**Comment by hamelsmu** [comment]
We are deprecating this bot in production, noted on the README as well.

**Pull request closed without merging**